components/prism-cooklang.js

```diff
@@ -0,0 +1,142 @@
+(function (Prism) {
+
+	var single_token_suffix = /[^{}@#\s]+/.source;
+	var multi_token_infix = /[^{}@#]+/.source;
+	var multi_token_suffix = /\{[^}#@]*\}/.source;
+
+	var multi_token = multi_token_infix + multi_token_suffix;
+
+	var timer_units = /(?:h|hours|hrs|m|min|minutes)/.source;
+
+	var amount_group_impl = {
+		pattern: new RegExp(/\{/.source
+			+ '(?:(?:'
+			+ /(?:[^{}|*%]+\*?)/.source // optional serving scale
+			+ '|'
+			+ /(?:(?:[^{}|*%]+\|)+[^{}|*%]+)/.source // serving alternatives
+			+ ')' + /(?:%[^{}|*%]+)?/.source //optional unit
+			+ ')?'
+			+ /\}/.source
+		),
+		inside: {
+			'amount': {
+				pattern: /([\{|])[^{}|*%]+/,
+				lookbehind: true,
+				alias: 'number',
+			},
+			'unit': {
+				pattern: /(%)[^}]+/,
+				lookbehind: true,
+				alias: 'symbol',
+			},
+			'servings-scaler': {
+				pattern: /\*/,
+				alias: 'operator',
+			},
+			'servings-alternative-seperator': {
+				pattern: /\|/,
+				alias: 'operator',
+			},
+			'unit-separator': {
+				pattern: /(?:%|(\*)%)/,
+				lookbehind: true,
+				alias: 'operator',
+			},
+			'punctuation': /[{}]/,
+		}
+	};
+
+	Prism.languages.cooklang = {
+		'comment': {
+			// [- comment -]
+			// -- comment
+			pattern: /\[-[\s\S]*?-\]|--.*/,
+			greedy: true,
+		},
+		'meta': { // >> key: value
+			pattern: />>.*:.*/,
+			inside: {
+				property: { // key:
```

Comment:
I know this is super nitpicky, but...

```suggestion
				'property': { // key:
```